parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -290,10 +298,18 @@ cleanUpContainerImages() {
             docker rmi ${images_to_delete[@]}
         fi
     }
-    export -f cleanUpHyperkubeImagesRun
     export -f cleanUpKubeProxyImagesRun
-    retrycmd_if_failure 10 5 120 bash -c cleanUpHyperkubeImagesRun
     retrycmd_if_failure 10 5 120 bash -c cleanUpKubeProxyImagesRun
+    echo $(date),$(hostname), endCleanUpKubeProxyImages
+}
+
+cleanUpContainerImages() {
+    # run cleanUpHyperkubeImages and cleanUpKubeProxyImages concurrently
+    export -f retrycmd_if_failure
+    export -f cleanUpHyperkubeImages
+    export KUBERNETES_VERSION
+    bash -c cleanUpHyperkubeImages &
+    cleanUpKubeProxyImages
```

Comment:
run in background as well

Reply:
CleanupContainerImages was already in a separate process (from cse_main.sh), so here we only need to make one of the funcs to background

Reply:
true, but nested structure is not easy to read/maintain, why not flatten them? Either run both in background and run foreground from cse_main.sh , or inline this method to cse_main.sh?

Reply:
Good point. Changed to call cleanupContainerImages directly then it will start two separate processes.